crates/testlang/testing/utils/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "testlang_testing_utils"
```

Comment:
is `testlang_testing_utils` used anywhere? If not I suggest removing it.
The matching Solidity APIs are only there for pragma extraction, and should be removed alltogether in #806